utils/qemu/Makefile

```diff
@@ -24,11 +24,12 @@ PKG_INSTALL:=1
 PKG_BUILD_FLAGS:=no-mips16
 
 PKG_BUILD_DEPENDS+=spice-protocol
+PKG_BUILD_DEPENDS+=python3/host
```

Comment:
This is intentional? Isnt not said in the commit description.

Reply:
yes this is intentional , without this line i can not build the package in https://hub.docker.com/r/openwrt/sdk . 

Qemu has a dependency only a build on a python package tomli  and adding this line was a simple way to fix .

Reply:
Shouldnt it be then in separate commit? How is this relevant to enabling qemu-qa for x86?

Reply:
In fact qemu can not be build in the docker image , i have the same issue with x86_64

Reply:
i removed the line , and let see if all checks will work .

Reply:
i removed the line , 3 check failed ... 

```
024-12-12T20:17:03.3282667Z python version: Python 3.9.2
2024-12-12T20:17:03.9865607Z mkvenv: Creating non-isolated virtual environment at 'pyvenv'
2024-12-12T20:17:04.6511595Z INFO:pip._vendor.distlib.util:changing mode of /builder/build_dir/target-x86_64_musl/qemu-9.1.0/build/pyvenv/bin/pip to 755
2024-12-12T20:17:04.6513343Z INFO:pip._vendor.distlib.util:changing mode of /builder/build_dir/target-x86_64_musl/qemu-9.1.0/build/pyvenv/bin/pip3 to 755
2024-12-12T20:17:04.6514538Z INFO:pip._vendor.distlib.util:changing mode of /builder/build_dir/target-x86_64_musl/qemu-9.1.0/build/pyvenv/bin/pip3.9 to 755
2024-12-12T20:17:05.3449476Z 
2024-12-12T20:17:05.3449908Z *** Ouch! ***
2024-12-12T20:17:05.3450129Z 
2024-12-12T20:17:05.3450275Z found no usable tomli, please install it 
2024-12-12T20:17:05.3450567Z 
2024-12-12T20:17:05.3450579Z 
2024-12-12T20:17:05.3639611Z make[2]: *** [Makefile:457: /builder/build_dir/target-x86_64_musl/qemu-9.1.0/.configured_0d81fcc86c4324447b68011a702bec08] Error 1
```

Reply:
i have the same error with the docker image .

Reply:
i move python modification in #25550